### PR TITLE
Feature/auth-backend-integration

### PR DESCRIPTION
**Pull request by govindappaarun**
Part of this PR 
- initialize project 
- adding backend default files, 
- api integration for auth  
- api integration for history

Not to verify: 
-   Components Folder, as its moved from other project
-  backend folder as its almost default ones

Ignore
  - contexts/
  - hooks/  

Must Check: 
   - pages/ * 
  - services/*

### CODE REVIEW

**Review by MayankKumar10 — APPROVED** [review]
Just a suggestion try to avoid creating long PR , Everything else looks good you can merge it.

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_JWT_SECRET="iloveyou"
```
Comment (vrx29) [inline .env:1]: you should put .env file in gitignore

**Review by vrx29 — APPROVED** [review]
Looks like just a setup for you app. You can merge it. I dont find any issues.

File: .env (hunk 1 above)
Comment (govindappaarun) [reply .env:1]: I will do this, need to fix hosting also accordingly

**Pull request merged**